.ahoy.yml

```diff
@@ -25,6 +25,7 @@ commands:
   build:
     usage: Build project.
     cmd: |
+      docker-compose build cli &&
```

Comment:
`depends-on` does not guarantee build order - with compose v2 this was more evident and resulted in failed builds with image inaccessible errors more frequently.

This ensures that the referenced multistage build image is available before attempting to bring the containers up.

```
------
 > [govcms-d9one_nginx internal] load metadata for docker.io/library/d9one:latest:
------
failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```